src/imageops/affine.rs

```diff
@@ -97,3 +97,115 @@ pub fn flip_vertical<I: GenericImage + 'static>(image:  &I)
 
     out
 }
+
+#[cfg(test)]
+mod test {
+    use super::{rotate90,rotate180,rotate270,flip_horizontal,flip_vertical};
+    use buffer::{ImageBuffer,Pixel,GrayImage};
+    use image::{GenericImage};
+
+    macro_rules! assert_pixels_eq {
+        ($actual:expr, $expected:expr) => ({
+            let actual_dim = $actual.dimensions();
+            let expected_dim = $expected.dimensions();
+
+            if actual_dim != expected_dim {
+                panic!("dimensions do not match. \
```

Comment:
I haven't written much rust either, so question I have is: why does this macro work inside `macro_rules!` but `assert!` doesn't?